src/components/Likes.jsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleByID, patchArticle } from "../api";
+
+const Likes = () => {
+  const { article_id } = useParams();
+  const [likes, setLikes] = useState(0);
+
+  useEffect(() => {
+    getArticleByID(article_id).then((article) => {
+      setLikes(article.votes);
+    });
+  }, [article_id]);
```

Comment:
Seems like you already have the number of votes on the article in the parent component, `SingleArticle`

Consider passing this value down instead of re-fetching the same data